Supporting Resources/v1.5/MarkDown/SAMM-1.5.1.md

```diff
@@ -3171,9 +3171,9 @@ Since code signing is not appropriate for everything, architects and developers
 | | **LSM1** | **LSM2** | **LSM3** |
 |:-------------|:-------------|:---------------|:-------------|:------------|
 | **Objective** | **Awareness of potential attack patterns.** | **Knowledge about impact of attack patterns to released Hardware and Software versions and variants.** | **Minimize negative impacts for customers.**|
```

Comment:
Hardware and Software --> hardware and software

Reply:
should be fixed with d7487364fe471c53230396fbc80f3a5b6ec25c08